keyboards/gmmk/gmmk2/p65/ansi/keymaps/via/config.h

```diff
@@ -0,0 +1 @@
+#define STM32_USB_USE_OTG1 TRUE
```

Comment:
THis needs a license header and an include guard (`#pragma once`).

Reply:
resolved.